Decred Wallet/Utils/DecredAddressURI.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  DecredAddressURI.swift
+//  Decred Wallet
+//
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+
+struct DecredAddressURI {
+    var amount: Double?
+    var address: String?
```

Comment:
- I think the `address` property should come before the `amount` property.
- `address` is assigned in `init`, should no longer be optional.

Reply:
resolved in `c32ac54`